crates/oracle_document/coordinator/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+pub mod oracle_document;
+use hdk::prelude::*;
+use shared_utils::get_authority_agent;
+
+#[hdk_extern]
+pub fn init(_: ()) -> ExternResult<InitCallbackResult> {
+    let authority_agent = get_authority_agent()?;
+    let my_pubkey = agent_info()?.agent_initial_pubkey;
+    let mut functions = BTreeSet::new();
+    let zome_name = zome_info()?.name;
+    if my_pubkey == authority_agent {
+        functions.insert((
+            zome_name.clone(),
+            "get_latest_oracle_document_ah_for_name".into(),
+        ));
+        functions.insert((
+            zome_name.clone(),
+            "get_latest_oracle_document_for_name".into(),
```

Comment:
Why have you added these cap grants?

Reply:
holoom_recipe -> holoom_recipe_execution ->*n. (all other zomes). the wrapper allows for the holoom_recipe_execution zome to call the oracle zome (bridge call)